### ISSUE TITLE
Why the validation loss is smaller than the training loss?

### ISSUE THREAD

**Issue by benjamin3344** [facebookresearch/NSVF#49]
I have replicated the NSVF datasets. The rendered views looked good. But why the training loss is always far larger than the validation loss?
